edgelet/contrib/config/linux/template.toml

```diff
@@ -0,0 +1,154 @@
+# ==============================================================================
```

Comment:
Where are the rest of the settings? In a separate toml file? 
How will they get merged? By iotedge apply?

Reply:
See TODO number 2

Reply:
Done.